stockAPI.js

```diff
@@ -0,0 +1,24 @@
+const request = require('request');
+
+var stockSymbols = ["AAPL", "MSFT","HSBA.L"];
+var str = "";
```

Comment:
global variables are a bad practice. define `str` inside the `getStockData` function